docs: update initial README docs (#33)
talos-systems/discovery-service: README.md

Node information is expired (if not updated) after 30 minutes.
Discovery service doesn't see actual node information, it only stores and updates encrypted blobs.
Discovery data should be submitted encrypted by the client, and service doesn't have the encryption key.

> The repo has been split into 3 parts(as they have differnt source code licenses), namely:

Reviewer: ```suggestion
The project has been split into 3 parts (as they have different source code licenses), namely:
```

> The repo has been split into 3 parts(as they have differnt source code licenses), namely:

Reviewer 2: ```suggestion
The repo has been split into 3 parts(as they have different source code licenses), namely:
```

> - The current repo

Reviewer: ```suggestion
- This repository
```

> - [Discovery client](https://github.com/talos-systems/discovery-client), contains the client code to interact with the server

Reviewer: ```suggestion
- [Discovery Client](https://github.com/talos-systems/discovery-client) contains the client code to interact with the server
```

> - [Discovery api](https://github.com/talos-systems/discovery-api/), provides various endpoints to access the server

Reviewer: ```suggestion
- [Discovery Service API](https://github.com/talos-systems/discovery-api/) provides gRPC API definition and cluster data protobuf data structures
```

> - [Discovery api](https://github.com/talos-systems/discovery-api/), provides various endpoints to access the server

Reviewer 2: I would probably suggest to rephrase this line to say that `discovery-api` contains protobuf definitions for the discovery service

Reviewer 2: oh, @smira was faster :D You can pick his suggestions for both

## Setup 

> All of the details to get started are present [in the make file](Makefile), start with `make help`.

Reviewer: ```suggestion
All of the details to get started are present [in the Makefile](Makefile), start with `make help`.
```

## Interacting

Once the application is running you can test the grpc functionality on the port 3000 and the http pages in browser on port 3001.

> To test the grpc calls, [install grpcurl from any of the ways mentioned here](https://github.com/fullstorydev/grpcurl#installation).

Reviewer 3: nit: since this is already a hyperlink, I'd trim the "from any of the ways here" part.  After all, the important part is to install grpcurl; it doesn't matter _how_ they install it, and the hyperlink provides the instructions within.
